Makefile

```diff
@@ -8,7 +8,7 @@
 #		TE: [TEST=] - Specifies the target test (only valid for tests, requires LI or PR to be specified)
 #		CM: [COMPILER=] - Specifies the compiler to use on x86. Defaults to gcc [gcc | clang].
 #		CO: [COPTIONS=] - Specifies compiler options on x86 [asan | tsan].
-#   PB: [PROBE=] - Specifies which debug probe to use on STM32F0xx. Defaults to cmsis-dap [cmsis-dap | stlink-v2].
+#       PB: [PROBE=] - Specifies which debug probe to use on STM32F0xx. Defaults to cmsis-dap [cmsis-dap | stlink-v2].
```

Comment:
Can you fix the spacing? Seems like the other lines should be converted to spaces.